packages/nextjs-template/hooks/useIFrameHeightAdjuster.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+const updateIFrameHeight = () => {
+  const iframes = document.querySelectorAll("iframe");
+  iframes.forEach((iframe, i) => {
+    if (iframe) {
+      const height = iframe!.contentWindow!.document.body.offsetHeight + 36;
```

Comment:
can you use a constant variable for 36 so we know where it comes from?

Reply:
also - does this work on mobile?

Reply:
Hmm, that magic number isn't actually needed anymore since we updated how the iframe stuff works.  Yes this works on mobile.